Tests/common_BioSQL.py

```diff
@@ -130,48 +130,58 @@ def check_config(dbdriver, dbtype, dbhost, dbuser, dbpasswd, testdb):
         raise MissingExternalDependencyError(message)
 
 
-def _do_db_create():
-    """Do the actual work of database creation.
+def _do_db_cleanup():
+    """Cleanup everything from TESTDB.
 
     Relevant for MySQL and PostgreSQL.
     """
-    # first open a connection to create the database
-    server = BioSeqDatabase.open_database(driver=DBDRIVER, host=DBHOST,
-                                          user=DBUSER, passwd=DBPASSWD)
 
-    if DBDRIVER == "pgdb":
+    if DBDRIVER == "psycopg2":
```

Comment:
Should this perhaps be ``if driver in ["psycopg2", "pgdb"]:`` here?

Reply:
I actually had considered this. I'll add pgdb there, just in case.